### ISSUE TITLE
Webpack watch isn't working on Windows for nested imports

### ISSUE THREAD

**Issue by gabrielenosso** [jtangelder/sass-loader#102]
Hi guys,
I created a repository in which you can check the bug: https://github.com/gabrielenosso/sass-loader-watch-problem .

I have a .js file which requires just one main SASS file: ./style.scss.
This file imports the file ./classes/button.scss.
The button.scss file imports another file: ./partials/_button.scss.

Practically, if you run the webpack dev server in hot mode (just executing "npm start"), it automatically reloads if you save the "style.scss" file (root of scss) or the ./classes/button.scss file (1st level of @import), but it doesn't reload if you save the ./partials/_button.scss file (2nd level of import).

It only happens on Windows. Could it be a bug related to '\' and '/' used for paths differently in Mac and Win? Or do you have any clue?

**Comment by jhnns** [comment]
Mhmm I don't have a working Windows setup to reproduce the error. Would be great if you could give me some insight. The sass-loader is not a huge project, so it should be fairly understandable.

[This is the line](https://github.com/jtangelder/sass-loader/blob/fff1c6857e2352f8cbc0230f8eae19ba30964b94/index.js#L249) that makes the included file known to webpack.

**Comment by newtonianb** [comment]
Just hit the exact same scernario/problem! Windows 8.1 definitely a bug
@jhnns @gabrielenosso

**Comment by jorrit** [comment]
Which Windows version do you use @gabrielenosso?

**Comment by gabrielenosso** [comment]
Windows 7, so it doesn't seem to be related to a particular version of Windows...

**Comment by newtonianb** [comment]
This is a huge problem, it seems also when ran through webpack-dev-server it can't even do past one level imports, the changes in imports don't trigger a webpack-dev-server change, I'm assuming it's related. Could someone please find a way to fix this, right now it's just unusable

**Comment by jhnns** [comment]
Do you use `npm link` or `bower link`?

**Comment by newtonianb** [comment]
Hi, not that I know of no

**Comment by strayiker** [comment]
```
// When files have been imported via the includePaths-option, these files need to be
// introduced to webpack in order to make them watchable.
function addIncludedFilesToWebpack(includedFiles) {
    includedFiles.forEach(function(filename) {
        self.dependency(path.normalize(filename));
    });
}
```
i add path.normalize() for dependencies and it works )
before: c:/repo/project
after: c:\repo\project

I hope it will help in solving the problem

**Comment by newtonianb** [comment]
Thank you @strayiker ! I do recall other gulp scripts having issues on windows due to paths. Where exactly do you put this function and make use of it?

**Comment by strayiker** [comment]
https://github.com/jtangelder/sass-loader/blob/master/index.js#L108-L112

**Comment by f0def** [comment]
Thank you @strayiker, [it](https://github.com/jtangelder/sass-loader/issues/102#issuecomment-125273005) helped me

**Comment by davidgilbertson** [comment]
Nice one @strayiker

**Comment by jhnns** [comment]
Will be fixed with the next major version.

**Comment by newtonianb** [comment]
Thanks alot! Just curiosity when is the next major version planned for?

**Comment by jhnns** [comment]
Fixed with 2.0.0
